OPERATOR.rst

```diff
@@ -640,14 +640,15 @@ downloads the tarball from the package registry and unpacks that tarball to the
 S3 bucket backing the Data Browser's CloudFront distribution.
 
 Typically, CC requests the deployment of a Data Browser instance on Slack,
-specifying the commit they wish to be deployed. After the system administrator
-approves that request, the operator merges the specified commit into one of the
-``ucsc/{atlas}/{deployment}`` branches and then pushes that branch to the
-``DataBiosphere/data-browser`` project on GitHub, and the ``ucsc/data-browser``
-project on the GitLab instance for the Azul ``{deployment}`` that backs the Data
-Browser instance to be deployed. For the merge commit title, SmartGit's default
-can be used, as long as the title reflects the commit (branch, tag, or sha1)
-specified by CC.
+specifying the commit (branch, tag, or sha1) they wish to be deployed. After the
```

Comment:
```suggestion
specifying the commit (tag, or sha1) they wish to be deployed. After the
```

When we discussed this in PL we agreed to support two cases (raw commit and tag). They should not specify a branch since that is too unreliable, since branches can easily change between when the request is made and the actions are actually taken by our system administrator and the operator.